src/TodoList/TodoItem/TodoItem.js

```diff
@@ -39,46 +37,46 @@ export class TodoItem extends React.Component {
   closeModifyTodo = () => {
     this.setState({
       editing: false,
-      currentId: 0,
-      currentText: ''
     })
   }
   
  render(){
-    const arrow = this.props.getArrowButton;
-    const status = this.props.todoStatus;
-    const todo = this.props.todo;
-    const {id, text} = todo;
+    const arrowRight = this.props.progressArrow;
+    const arrowLeft = this.props.regressArrow;
 
-   if(this.state.editing) {
-      return(
-        <div className="todo-item">
-          <input
-          type="text"
-          className="todo-input"
-          onChange={this.modifyTodoHandler}
-          value={this.state.currentText}/>
-          <Button
-          clickHandler={this.confirmModifyTodoHandler}
-          classExtra="btn-ok"
-          icon={<FaCheck />}/>
-        </div>
+  if(this.state.editing) {
+    return(
```

Comment:
This could be two separate components, one for normal state, another for editing state.